plugins/duck.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: duck
+spec:
+  version: v0.0.1
+  homepage: https://github.com/n3wscott/kubectl-duck
+  shortDescription: List custom resources with ducktype support
+  description: |
+    This plugin allows you to list and get custom resources based on ducktype
+    labels on CRDs.
+  caveats: |
+    * Ducktype labels are currenly hardcoded to known types from Knative.
```

Comment:
Is that leading star intentional?

Reply:
I think it is a list. but has only one point.

Reply:
that's fine I think.